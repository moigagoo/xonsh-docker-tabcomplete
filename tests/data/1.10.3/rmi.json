{
        "arguments": [
                "-f", "--force",
                "--help",
                "--no-prune"
        ],
        "commands": []
}
