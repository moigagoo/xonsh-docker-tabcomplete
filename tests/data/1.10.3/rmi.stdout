
Usage:	docker rmi [OPTIONS] IMAGE [IMAGE...]

Remove one or more images

  -f, --force        Force removal of the image
  --help             Print usage
  --no-prune         Do not delete untagged parents
